core/src/services/routing.js

```diff
@@ -288,6 +288,12 @@ const buildRoute = (node, path, params) =>
     : buildRoute(node.parent, `/${node.parent.pathSegment}${path}`, params);
 
 export const handleRouteChange = async (path, component, node, config) => {
+  if (
+    window.location.href.indexOf('access_token') !== -1 ||
```

Comment:
1) Can we do this check a bit more specific? At least indexOf('access_token=').
2) Can we make it configurable? Like list of regex expressions for paths to be ignored by Luigi routing? That way the Luigi app owner would have full control over it.

Reply:
Yes, thats a good idea :-)